frontend/src/components/RepositoryForm.vue

```diff
@@ -207,6 +209,9 @@ import {
 
 const FILE_REQUIRED_PLACEHOLDER = "{{DB_NAME}}, {{VERSION}}, {{TYPE}}";
 const SCHEMA_REQUIRED_PLACEHOLDER = "{{DB_NAME}}";
+const FILE_OPTIONAL_FOLDER_WILDCARD = "*, **";
```

Comment:
nit: I think we have been using "directory" instead of "folder" throughout our codebase, unless there is a specific reason, I'd suggest the following
```suggestion
const FILE_OPTIONAL_DIRECTORY_WILDCARD = "*, **";
```

Reply:
LGTM.